contracts/VeJoeStaking.sol

```diff
@@ -87,17 +93,26 @@ contract VeJoeStaking is Initializable, OwnableUpgradeable {
         uint256 _boostedDuration,
         uint256 _maxCap
     ) public initializer {
+        UPPER_LIMIT_BOOSTED_DURATION = 365 days;
+        UPPER_LIMIT_MAX_CAP = 100000;
```

Comment:
Can we define `UPPER_LIMIT_MAX_CAP`, `UPPER_LIMIT_BOOSTED_DURATION` and `PRECISION` as immutable variables and assign them in a constructor instead?

Or, we don't use upper snake case.